objects/ogarbagecan/Create_0.gml

```diff
@@ -0,0 +1,36 @@
+/// @description Insert description here
+// You can write your code in this editor
+
+event_inherited();
+
+hp = 30;
```

Comment:
I believe this variable is inherited, but if not please don't use abbreviations and use ```hitPoints``` instead.

Reply:
It is, I change the value after inheritance to differentiate it from other destructible objects